velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -447,11 +444,14 @@ bool PrestoQueryRunner::isSupported(const exec::FunctionSignature& signature) {
   // cast-to or constant literals. Hyperloglog can only be casted from varbinary
   // and cannot be used as the type of constant literals. Interval year to month
   // can only be casted from NULL and cannot be used as the type of constant
-  // literals.
+  // literals. Json requires special handling, because Presto requires Json
+  // literals to be valid Json strings, and doesn't allow creation of Json-typed
+  // HIVE columns.
   return !(
       usesTypeName(signature, "interval year to month") ||
       usesTypeName(signature, "hugeint") ||
-      usesTypeName(signature, "hyperloglog"));
+      usesTypeName(signature, "hyperloglog") ||
```

Comment:
Hi @bikramSingh91, it looks like this change was accidentally included as part of https://github.com/facebookincubator/velox/pull/11276 that was the base diff of the current PR. When the current PR was exported, the base diff was exported as a commit together. I've rebased the current PR so that the base commit is removed.